### ISSUE TITLE
Incorrect progress messages in multi-pass rendering from appleseed.cli

### ISSUE THREAD

**Issue by dictoon** [appleseedhq/appleseed#864]
We get things like

```
2015-09-22T11:39:46.395100Z <011>  2321 MB info    | rendering, 128.5% done
2015-09-22T11:39:46.422126Z <009>  2321 MB info    | rendering, 129.2% done
2015-09-22T11:39:46.578275Z <002>  2321 MB info    | rendering, 129.9% done
```

This is due to `appleseed::cli::ProgressTileCallback` not being aware that a new pass has started. We should have a pass callback (`renderer::IPassCallback`) to get notified about passes being started/ended.

**Comment by dictoon** [comment]
Will be solved once #971 is done.

**Comment by dictoon** [comment]
Fixed.
